crates/sncast/src/helpers/interactive.rs

```diff
@@ -0,0 +1,114 @@
+use crate::helpers::config::get_global_config_path;
+use anyhow::{Context, Result};
+use camino::Utf8PathBuf;
+use configuration::search_config_upwards_relative_to;
+use dialoguer::theme::ColorfulTheme;
+use dialoguer::Select;
+use std::env::current_dir;
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::Path;
+use toml_edit::{DocumentMut, Item, Table, Value};
+
+pub fn ask_to_add_as_default(account: &str) -> Result<()> {
+    let mut options = Vec::new();
+
+    if let Ok(global_path) = get_global_config_path() {
+        let option = format!(
+            "Yes, global default account ({}).",
+            to_tilde_path(global_path.as_str())
+        );
+        options.push(option);
+    }
+
+    if let Ok(current_path) = current_dir() {
+        let current_path_utf8 = Utf8PathBuf::from_path_buf(current_path.clone())
+            .expect("Failed to convert current directory to Utf8PathBuf");
```

Comment:
I don't think `expect` is a good approach here. I'd either bubble the error or handle it by not displaying the local option.